### ISSUE TITLE
Add validation for various input type when generating project

### ISSUE THREAD

**Issue by retzd-tech** [MicrofrontendGenerator/generator#7]
Condition:
Currently when devs are going to generate microfrontend, there will be some inputs like package name, version, description, etc, and all of those still not validated yet, example, input project initial version, you can put any string, not "0.0.1" format. so you can also use regex for this question.

Expected:
Add validation. how to handle error and what to say to dev when error occured when providing information.

Hint :
- you can add a function to validate in question.js

**Comment by srijit2002** [comment]
@retzd-tech can you please explain about this a little bit?

**Comment by retzd-tech** [comment]
> @retzd-tech can you please explain about this a little bit?

Yes, currently when devs are going to generate microfrontend, there will be some inputs like package name, version, description, etc.

and all of those still not validated yet, example, input project initial version, you can put any string, not "0.0.1" format. so you can also use regex for this question.

and then after validation. how to handle error and what to say to dev when error occured.

**Comment by srijit2002** [comment]
@retzd-tech ok. So what are those file I need to change?

**Comment by retzd-tech** [comment]
> @retzd-tech ok. So what are those file I need to change?

Anywhere where it requires a change, please understand the code first.
